spanner/tests/system/test_system.py

```diff
@@ -728,6 +728,33 @@ def _unit_of_work(transaction, test):
 
         return session, committed
 
+    def test_read_with_multiple_keys_index(self):
+        ROW_COUNT = 10
+        EXTRA_DDL = ['CREATE INDEX name ON contacts(first_name, last_name)']
+        MY_COLUMNS = self.COLUMNS[1], self.COLUMNS[2]
+        pool = BurstyPool()
+
+        temp_db = Config.INSTANCE.database(
+            'test_read_multi_keys_index', ddl_statements=DDL_STATEMENTS + EXTRA_DDL,
```

Comment:
Because quotas prevent us from using per-CI-run instances, this test needs to mangle the database ID with `unique_resource_id`.

Another, maybe better strategy might be to just add the `CRETATE INDEX` DDL to the "stock" set in `tests/_fixtures.py`.

Reply:
I either get
`=================================== FAILURES ===================================
______________ TestSessionAPI.test_read_with_multiple_keys_index _______________
Traceback (most recent call last):
  File "/home/aw/projects/google-cloud-python/spanner/tests/system/test_system.py", line 738, in test_read_with_multiple_keys_index
    session, committed = self._set_up_table(ROW_COUNT)
  File "/home/aw/projects/google-cloud-python/spanner/tests/system/test_system.py", line 716, in _set_up_table
    retry(db.reload)()
  File "/home/aw/projects/google-cloud-python/spanner/.nox/sys-2-7/local/lib/python2.7/site-packages/test_utils/retry.py", line 205, in wrapped_function
    raise BackoffFailed()
BackoffFailed
----------------------------- Captured stdout call -----------------------------
_has_all_ddl. Trying again in 1 seconds...
_has_all_ddl. Trying again in 2 seconds...
_has_all_ddl. Trying again in 4 seconds...
_has_all_ddl. Trying again in 8 seconds...
============================= 41 tests deselected ==============================`
or
`
=================================== FAILURES ===================================
______________ TestSessionAPI.test_read_with_multiple_keys_index _______________
Traceback (most recent call last):
  File "/home/aw/projects/google-cloud-python/spanner/tests/system/test_system.py", line 737, in test_read_with_multiple_keys_index
    operation.result(90)
  File "/home/aw/projects/google-cloud-python/spanner/.nox/sys-2-7/local/lib/python2.7/site-packages/google/gax/__init__.py", line 595, in result
    if not self._poll(timeout).HasField('response'):
  File "/home/aw/projects/google-cloud-python/spanner/.nox/sys-2-7/local/lib/python2.7/site-packages/google/gax/__init__.py", line 705, in _poll
    return retryable_done_check()
  File "/home/aw/projects/google-cloud-python/spanner/.nox/sys-2-7/local/lib/python2.7/site-packages/google/gax/retry.py", line 143, in inner
    raise exc
RetryError: RetryError(Retry total timeout exceeded with exception, caused by Deadline Exceeded)
============================= 41 tests deselected ==============================
`